### ISSUE TITLE
A11y_AzureDataStudio_ClusterSettings_ScreenReader:NVDA announces unnecessarily "clickable" for the non actionable text.

### ISSUE THREAD

**Issue by vinodbk197** [microsoft/azuredatastudio#7598]
"[Check out Accessibility Insights!](https://accessibilityinsights.io) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

**GitHubTags:**#A11y_AzureDataStudio;#A11yMAS; #A11yTCS;#SQL Azure Data Studio;#WCAG1.3.1;#DesktopApp;#NVDA;#Win32;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.12.0-insider (user setup)
Commit: 32df727
Date: 2019-10-08T19:18:43.743Z
VS Code: 1.38.0
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362

**Additional Details:**
NVDA Version:2019.2

**Repro Steps:**

1. Install /Launch the application Azure data studio.
2. Navigate to connections and click on more action button.
3. Select “Deploy SQL Server big data cluster” from more action drop down.
4. Click on “Deploy SQL Server big data cluster”.
5. Check the check box I accept in the “select the deployment option” screen and click on
select button.
6. Select any block and click on next button and again click on next button.
7. " Cluster settings" screen is opened
8. Start the NVDA.
9. Navigate on screen by using down arrow key and observe NVDA announces unnecessarily "clickable" for the non actionable text.

**Actual:**
NVDA announces unnecessarily "clickable" for the non actionable text.

**This issue is not observed in JAWS.**

**Expected:**
NVDA should not announce unnecessarily "clickable" for the non actionable text.

**Same issue observed for the below scenario.**
**1.services sittings:**
1.	Install /Launch the application Azure data studio.
2.	Navigate to connections and click on more action button.
3.	Select “Deploy SQL Server big data cluster” from more action drop down.
4.	Click on “Deploy SQL Server big data cluster”.
5.	Check the check box I accept in the “select the deployment option” screen and click on select button.
6.	Select any block and click on next button and again click on next button.
7.	Enter password and confirm password and click on next button.
8.	Verify that all the elements of " Cluster settings" are accessible and MAS Compliant.

**2.Summary:**
1.	Install /Launch the application Azure data studio.
2.	Navigate to connections and click on more action button.
3.	Select “Deploy SQL Server big data cluster” from more action drop down.
4.	Click on “Deploy SQL Server big data cluster”.
5.	Check the check box I accept in the “select the deployment option” screen and click on select button.
6.	Select any block and click on next button and again click on next button.
7.	Enter password and confirm password and click on next button and again click on next button.
8.	Verify that all the elements of " Summary" are accessible and MAS Compliant.

**User Impact:**
Screen reader user will get confused and try to activate the control for the non actionable text. If the screen reader announces clickable.

**Attachment for Reference:**
[7598_A11y_AzureDataStudio_ClusterSettings_ScreenReader.pptx](https://github.com/microsoft/azuredatastudio/files/3712392/7598_A11y_AzureDataStudio_ClusterSettings_ScreenReader.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by alanrenmsft** [comment]
should have been fixed by https://github.com/microsoft/azuredatastudio/commit/52de2b475108918159a6dfea59efa7407663afa7

**Comment by vinodbk197** [comment]
Verified the issue on: 
Azure Data Studio
Version: 1.15.0-insider (user setup)

The issue is resolved,Hence closed this issue.
Please refer the below attachment:
[7598_Closed.pptx](https://github.com/microsoft/azuredatastudio/files/4086372/7598_Closed.pptx)

**Comment by fsteffi** [comment]
#Azure Data Studio - New Features_win32;

**Comment by Yash14j** [comment]
#A11Ysev2;#Closed;
